vertx-web-api-service-example/pom.xml

```diff
@@ -0,0 +1,159 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>io.vertx</groupId>
+  <artifactId>vertx-web-api-service-example</artifactId>
+  <version>3.6.0-SNAPSHOT</version>
+
+  <name>asd</name>
+
+  <properties>
+    <main.verticle>io.vertx.examples.webapiservice.WebApiServiceExampleMainVerticle</main.verticle>
+  </properties>
+
+
+  <dependencies>
+    <dependency>
+      <groupId>io.vertx</groupId>
+      <artifactId>vertx-core</artifactId>
+      <version>3.6.0-SNAPSHOT</version>
```

Comment:
Please use `<version>${project.version}</version>` here and below (it minimizes changes when upgrading versions)